### DISCUSSION TITLE
Divider does not display on Sakai

### DISCUSSION THREAD

**Discussion by msn444** [primefaces/community#999] [PrimeVue Templates] [answered]
This seems to be a similar issue to https://github.com/primefaces/primevue/issues/4647, but I'm unable to relate the Poseidon solution to Sakai. @tugcekucukoglu can you advise how to fix in a Sakai project?

**Comment by tugcekucukoglu** [comment, marked as answer]
Issue fixed for the 3.48.1.

Thanks for your report!

**Reply by msn444** [reply]
Thank you. However, in updating primevue to 3.48.1, many styling elements get messed up. You can reproduce this by cloning the Sakai demo and run `npm i primevue@latest`. What else do I need to do?
